### DISCUSSION TITLE
Error after terraform init terraform apply command

### DISCUSSION THREAD

**Discussion by sandeepdevacc** [serverpod/serverpod#446] [Q&A]
aws_launch_configuration.serverpod: Creating...
╷
│ Error: couldn't find resource
│ 
│   with aws_launch_configuration.serverpod,
│   on instances.tf line 19, in resource "aws_launch_configuration" "serverpod":
│   19: resource "aws_launch_configuration" "serverpod" {

MY ws_launch_configuration file snippet 

resource "aws_launch_configuration" "serverpod" {
  name_prefix = "${var.project_name}-"
  image_id    = var.instance_ami
  #   image_id = data.aws_ami.amazon-linux.id
  instance_type = var.instance_type
  user_data     = templatefile("init-script.sh", { runmode = "production" })

**Comment by yahu1031** [comment]
Can you provide us with a little more info please, like all the data in the console you see? @sandeepdevacc

**Comment by sandeepdevacc** [comment]
@yahu1031 Thanks 
![image](https://user-images.githubusercontent.com/118791435/205569151-b4f58fba-3e94-4f69-9210-51152729f3dd.png)

**Comment by yahu1031** [comment]
Okie what are the steps you have followed till that? @sandeepdevacc

**Comment by sandeepdevacc** [comment]
@yahu1031  all the steps that are provided by the Serverpod doc 

- I have my domain with me 
- I have 2 SSL certificates with me 
- I pushed my code to GitHub
- Changed my secret on GitHub
- and after all, this changed the .tfvars file with my hosted zone id, domain, and certificates and ran the command terraform init and terraform apply commands, and typed yes to continue 
- and got the above error

**Comment by yahu1031** [comment]
I was thinking, you are missing something.

Did you do this?
```sh
export TF_VAR_DATABASE_PASSWORD_PRODUCTION="<MY PRODUCTION PASSWORD>"
```

I would recommend watching this https://www.youtube.com/watch?v=zx2eh3t_98Q by CodeX

**Comment by sandeepdevacc** [comment]
@yahu1031 yes am followed the same video and official doc of Serverpod.
and I have set the DB password as well, But facing the same error as mentioned.

**Comment by sandeepdevacc** [comment]
@yahu1031 Update solved the above error but now another error occurred when I connected to Postico then its shows the below Hostname not found.
error 
![image](https://user-images.githubusercontent.com/118791435/206185298-d0fbcba2-809b-41f4-a4ab-3a3876902f59.png)

**Reply by yahu1031** [reply]
It says connection lost. Check out the domain you have.

**Comment by sandeepdevacc** [comment]
@yahu1031  the server is up and running,
![image](https://user-images.githubusercontent.com/118791435/206368363-12e27c53-870e-458d-b577-20068a02ebfb.png)

but when connecting to Postico 2 its shows the below error : 
connection to server at "database.devsandeep.shop" (34.102.1XX.XX0), port 5432 failed: Operation timed out
	Is the server running on that host and accepting TCP/IP connections?

**Reply by yahu1031** [reply]
Try ping command in terminal `ping database.devsandeep.shop`

**Reply by yahu1031** [reply]
Looks like the server is not up from serverpod code. `api.devsandeep.shop` this doesn't work too.

**Reply by sandeepdevacc** [reply]
@yahu1031 @yahu1031 
![image](https://user-images.githubusercontent.com/118791435/206411385-d0cd9210-1a01-4a77-a362-496ea87f1738.png)


![image](https://user-images.githubusercontent.com/118791435/206377886-f84689df-49df-4758-b1fd-8ab7e4ff3959.png)
